lib/next_rails/bundle_report/cli.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+class NextRails::BundleReport::CLI
+  def initialize(argv)
+    validate_arguments(argv)
```

Comment:
JFI: I'm thinking that more than validate you can follow RubyCritic approach, to have default params, etc. 

https://github.com/whitesmith/rubycritic/blob/main/lib/rubycritic/configuration.rb

Reply:
I can do some work on the argument side, yes.
About the tests, I wonder if since I'm not changing the gem's behavior the current test would be enough for it.

Reply:
I'm not sure if we should make any changes to the validation at this stage, as it seems to stray from our original objective.

Reply:
So I would consider removing it entirely and only add a `TODO`